summarizer/src/main/java/fish/payara/internal/tcksummarizer/TxtParser.java

```diff
@@ -0,0 +1,149 @@
+ *    "Portions Copyright [year] [name of copyright owner]"
+ *
+ *    Contributor(s):
+ *    If you wish your version of this file to be governed by only the CDDL or
+ *    only the GPL Version 2, indicate your decision by adding "[Contributor]
+ *    elects to include this software in this distribution under the [CDDL or GPL
+ *    Version 2] license."  If you don't indicate a single choice of license, a
+ *    recipient has the option to distribute your version of this file under
+ *    either the CDDL, the GPL Version 2 or to extend the choice of license to
+ *    its licensees as provided above.  However, if you add GPL Version 2 code
+ *    and therefore, elected the GPL Version 2 license, then the option applies
+ *    only if the new code is made subject to such option by the copyright
+ *    holder.
+ */
+
+package fish.payara.internal.tcksummarizer;
+
+import java.io.File;
+import java.util.Scanner;
+
+import java.util.Collection;
+import java.util.function.Consumer;
+
+public class TxtParser {
+
+    private int tests = 0;
+    private int failures = 0;
+    private int errors = 0;
+    private int skipped = 0;
+    
+  public String parseSummaryReport(String inputFilePath, String testSuiteName) {
+
+      String result = new String();
+            
+      try {
+
+        File file = new File(inputFilePath);
+        Scanner sc = new Scanner(file);
+
+        while (sc.hasNextLine())
+        {
+            String currentLine = sc.nextLine();
+            
+            if (!currentLine.isEmpty()) 
+                {
+                    tests ++;
+                }
+            if (currentLine.contains("Failed.")) 
+                { 
+                    failures ++; 
+                }
+       }
+        
+        result = "### " + testSuiteName + "\n \nCompleted running " + tests + " tests\n";
+        result +=  "Number of tests failed " + failures +  "\n";
+        result +=  "Number of tests with errors " + errors +  "\n";
+        result +=  "Number of tests skipped " + skipped +  "\n";
+
+        System.out.println(result);
+        
+        
+        
+      } catch (Exception e) {
+              e.printStackTrace();
+      }
+      
+      return result;
+  }
+  
+
+  public String parseTestSetReport(Collection<String> inputFilePaths, String testSuiteName) {
+
+      String result;
+      
+      inputFilePaths.forEach(new Consumer<String>() {
+          @Override
+          public void accept(String inputFilePath) {
+              try {
+                  
+                  // pass the path to the file as a parameter
+                  File file = new File(inputFilePath);
+                  Scanner sc = new Scanner(file);
+                  
+                  while (sc.hasNextLine()) 
+                  {
+                      String currentLine = sc.nextLine();
+                      
+                      if (currentLine.contains("Tests run: "))
+                      {
```

Comment:
For new files, I suggest format them (Java Coding Standard says opening brace on the same line).